packages/workspace/src/Workspace.ts

```diff
@@ -0,0 +1,208 @@
+import uuidv4 from 'uuid/v4';
```

Comment:
export const guid = (n = 3) => {
  const possible = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
  let text = '';
  for (let i = 0; i < n; i++) {
    text += possible.charAt(Math.floor(Math.random() * possible.length));
  }
  return text;
};